JetAnalyzers/python/addAlgorithm.py

```diff
@@ -618,7 +621,7 @@ def addAlgorithm(process, alg_size_type_corr, Defaults, reco, doProducer):
         if correctl1 or correctl2l3:
             jetPtEta.src = corrLabel
     elif reco:
-        process.load('Configuration.StandardSequences.Geometry_cff')
+        #process.load('Configuration.StandardSequences.Geometry_cff')
```

Comment:
If you don't need this lines anymore can it be removed? Same with line 612 an 614.